### PR TITLE
	feat: split service name and service version

### PR DESCRIPTION
**Pull request by skygeo**
Split the service name and service version.
Makes Ground Control aware of projects, service and version.
Update to the deployment table range key, from cluster to a json with cluster and version. 
This is a breaking change that will require the replacement of dynamodb table.

### CODE REVIEW

**Comment by paulci** [comment]
LOKTM

@gjtempleton Would you like to lend a second pair of eyes>

File: tests/test_scheduler.py [hunk 1]
```diff
@@ -412,7 +420,7 @@ def test_scheduler_scales_down_deactivates_empty_lb_deployements(
     service_definition['loadbalancer_config'] = None
     flyby = mock_scheduler_flyby(service_definition)
 
-    # Start a deploy
+    # `Start a deploy
```
Comment (gjtempleton) [inline tests/test_scheduler.py]: Stray punctuation here.

**Comment by gjtempleton** [comment]
LGTM

**Comment by balintzs** [comment]
Please add description about what these changes are, why they are necessary, and what impact they have.

**Pull request merged**